core/kazoo_proper/src/pqc_cb_api.erl

```diff
@@ -21,7 +21,7 @@
              ]).
 
 -define(API_BASE, "http://localhost:8000/v2").
--define(API_KEY, <<"df52943da7119a3fc901dcadba313c0689d6d883dd26f5fa84abbc44df6c30d1">>).
+-define(API_KEY, <<"d056f9d536914499dd1e8cf63593be4ea333368adf630583a38db30b1877a710">>).
```

Comment:
that was my thought eventually, esp once this can be a CI-like or make-busy-like suite